docs/knn/index.md

```diff
@@ -112,3 +115,98 @@ GET my-index/_search
   }
 }
 ```
+
+
+## Custom scoring
+
+The [previous example](#mixing-queries) shows a search that returns fewer than `k` results. If you want to avoid this situation, KNN's custom scoring option lets you essentially invert the order of events.
+
+First, add another index:
+
+```json
+PUT my-knn-index-2
+{
+  "settings": {
+    "index.knn": true
+  },
+  "mappings": {
+    "properties": {
+      "my_vector": {
+        "type": "knn_vector",
+        "dimension": 2
+      },
+      "color": {
+        "type": "keyword"
+      }
+    }
+  }
+}
+```
+
+If you *only* want to use KNN's custom scoring, you can omit `"index.knn": true`, but you lose the ability to perform standard KNN queries on the index. The benefit of this approach is faster indexing speed and lower memory usage.
```

Comment:
How about we change to below?

The benefit of this approach is **minimal search latencies as the graphs are prebuilt with provided vectors during indexing and then cached for later queries.**

Reply:
So the goal here was actually to talk about the benefit of a `false` setting. I can tweak the wording there since that wasn't clear.